File: Lab2/app/src/main/java/com/example/firstjavaapp/models/Season.java
package com.example.firstjavaapp.models;

import java.util.ArrayList;

public class Season {
    public ArrayList<DocSeason> docs;
    public int total;
    public int limit;
    public int page;
    public int pages;

    @Override
    public String toString(){
        return String.valueOf(this.docs.get(0).episodes.get(0).name) + " , всего: " + this.total;
    }
}

File: Lab_2_NEW/app/src/main/java/com/example/lab_2_new/models/Season.kt
package com.example.lab_2_new.models

class Season {
    var docs: ArrayList<DocSeason>? = null
    var total = 0
    var limit = 0
    var page = 0
    var pages = 0

    override fun toString(): String {
        return java.lang.String.valueOf(docs!![0].episodes!![0].name)
            .toString() + " , всего: " + total
    }
}